### PR TITLE
chore: increase grpc max call send & receive msg size

### PR DESCRIPTION
**Pull request by bsushmith**
* Increase the grpc max call send & receive msg size to `32MB` from default `4MB`.

### CODE REVIEW

File: internal/server/server.go [hunk 1]
```diff
@@ -100,7 +104,14 @@ func Serve(
 
 	headerMatcher := makeHeaderMatcher(config)
 
-	grpcConn, err := grpc.DialContext(grpcDialCtx, config.grpcAddr(), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	grpcConn, err := grpc.DialContext(
+		grpcDialCtx,
+		config.grpcAddr(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+		grpc.WithDefaultCallOptions(
+			grpc.MaxCallRecvMsgSize(GRPCMaxClientSendSize),
```
Comment (StewartJingga) [inline internal/server/server.go:112]: Do we need to increase RecvMsg as well? All of our insert APIs do not seem to need it.
wdyt?

File: internal/server/server.go [hunk 2]
```diff
@@ -30,6 +30,10 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+const (
+	GRPCMaxClientSendSize = 32 << 20
```
Comment (StewartJingga) [inline internal/server/server.go:34]: should we make it configurable as well ? wdyt?

Comment (bsushmith) [reply internal/server/server.go:34]: Yes. I think we should be able to configure it also if needed.

File: internal/server/server.go (hunk 1 above)
Comment (bsushmith) [reply internal/server/server.go:112]: Not sure about this. 
Increased the `RecvMsg` as well, as asset payload can be huge - but haven't faced this scenario yet.

File: internal/server/server.go (hunk 2 above)
Comment (bsushmith) [reply internal/server/server.go]: made both values to be configurable

**Review by sudo-suhas — APPROVED** [review]

**Pull request merged**